frontend/src/modules/contributor/components/details/contributor-details-notes.vue

```diff
@@ -1,14 +1,97 @@
 <template>
-  <app-member-view-notes class="!pt-0" :member="props.contributor" />
+  <div v-if="hasPermission(LfPermission.noteCreate)" v-bind="$attrs">
+    <lf-note-editor
+      :properties="{ members: [props.contributor.id] }"
+      @created="fetchNotes()"
+    />
+  </div>
+  <div v-else-if="!notes.length" class="w-full text-gray-400 italic text-small">
```

Comment:
I think you are missing an empty state when there are no notes
![Screenshot 2024-06-18 at 12 01 31](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/bb82237d-ddcf-425d-873f-62c41b60e85f)